guides/v2.2/cloud/env/restore-environment.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Restore an environment
+group: cloud-guide
+---
+
+If you encounter issues in your environment, but you do not have a [valid snapshot]({{ page.baseurl }}/cloud/project/project-webint-snap.html), you can try restoring your environment in on of the following ways:
+
+-  SSH
+-  Git
+
+## SSH
+
+If you have not created a snapshot but you can access the environment using SSH, follow these steps to restore your environment:
+
+-  Disable configuration management
```

Comment:
Should they be numbered? Is the order important?

Reply:
Yes, I think so. The original KB article numbers them, so I'll update it.